apps/src/lib/kits/maker/util/boardUtils.js

```diff
@@ -19,16 +19,27 @@ export const BOARD_TYPE = {
 export function detectBoardTypeFromPort(port) {
   let boardType = BOARD_TYPE.OTHER;
   if (port) {
-    const vendorId = port.vendorId ? parseInt(port.vendorId, 16) : null;
-    const productId = port.productId ? parseInt(port.productId, 16) : null;
-    if (vendorId === ADAFRUIT_VID && productId === CIRCUIT_PLAYGROUND_PID) {
+    const vendorId = port.vendorId;
+    const productId = port.productId;
```

Comment:
super nit: there's an ES6 shorthand for this using object destructuring: `const {vendorId, productId} = port;`